src/app/components/progress-indicator/index.html

```diff
@@ -1,20 +1,266 @@
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div class="sky-margin-stacked-separate">
+					<img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-8.png" />
+					<div class="caption caption-danger sky-text-danger">Don't revert a finished state back into a waterfall to edit it.</div>
+				</div>
+                <div>
+					<img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-9.png" />
+					<div class="caption caption-success sky-text-success">Do divide pieces of the finished state into groups that can edited in isolation.</div>
+				</div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+    </sky-demo-page-content>
+        
+    
+        
+    <sky-demo-page-content sectionHeading="Anatomy">
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/progress-indicator/waterfall-progress-anatomy.png">
+            <sky-docs-anatomy-item anatomyNumber="1">Step indicator</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="2">Step label</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="3">Previous step action</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="4">Current step action</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item isOptional="true" anatomyNumber="5">Waterfall title</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item isOptional="true" anatomyNumber="6">Step details</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item isOptional="true" anatomyNumber="7">"Start over" action</sky-docs-anatomy-item>
+        </sky-docs-anatomy>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Options">
+        <h3>Waterfall title</h3>
+        <div class="sky-text-block">
+            <p>It is usually best to include a title to reflect the context of the overall task. The title does not change as the user progresses through the waterfall steps.</p>
+        </div>
+        
+        <h3>Step details</h3>
+        <div class="sky-text-block">
+            <p>You can provide additional details about a completed step, limited to a single field. Don’t go overboard; users can reopen the modal to review their choices if they need.</p>
+        </div>
+        
+        <h3>"Start over"</h3>
+        <div class="sky-text-block">
+            <p>You can include an optional “start over” button that allows the user to erase all of the saved information for the waterfall process. This can return the user to the first step in the waterfall or, if some decision in the task occured before the waterfall, to another previous point in the workflow.</p>
+        </div>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Behavior & states">
+        <div class="sky-text-block">
```

Comment:
This section has a lot of text, is it possible to trim it down and show examples?

Reply:
Resolving based on other comment threads...